### PR TITLE
Refine content on help docs landing page

### PR DESCRIPTION
**Pull request by nkkl**
- [x] simplified language
- [x] reprioritized reasons to use static sites

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -3,28 +3,27 @@ permalink: /
 title: About Federalist
 ---
```
Comment (nkkl) [inline pages/index.md]: We should add a simple, snappy tagline here as a TL;DR.

File: pages/index.md [hunk 2]
```diff
@@ -3,28 +3,27 @@ permalink: /
...
-Static websites store content in text files. The Federalist editor provides a way for editing these files without needing to write code.
+- **Update content without writing code** Static websites store content in simple text files that anyone on the team can edit. The Federalist editor provides a way for editing these files without needing to write code.
 
+- **Easy to host and maintain** It’s very easy to host static website files. You can easily take advantage of scaling services like Amazon S3 where you pay only for what you use.
 
-## What sites are possible with Federalist
+- **Fully customizable** Designers and developers can customize static websites into whatever form they want without having to strip away CMS-provided features and defaults. Static sites can be used to build many different things, from organization homepages or intranets to pre-generated web application program interfaces (API).
 
-Federalist provides ready-to-use templates for common websites, as well as support for custom website templates based on Jekyll.
+## What can I build with Federalist?
```
Comment (nkkl) [inline pages/index.md]: Would love to get hair more detailed about the types of things you can build and provide some concrete examples — e.g. "a simple homepage for your organization", "a dynamic API-backed service", etc.

File: pages/index.md [hunk 3]
```diff
@@ -37,6 +36,4 @@ Here are a few examples of Federalist websites in production:
 
 ## Getting started
 
-Federalist is currently under active development. We are working with a select list of partners to test the platform and launch new websites.
```
Comment (nkkl) [inline pages/index.md]: I moved this content to the top of the page as an immediate FYI. Not diehard about it.

File: pages/index.md (hunk 1 above)
Comment (wslack) [inline pages/index.md]: help me think of one!

**Pull request merged**